src/kernel/src/fs/dev/dipsw.rs

```diff
@@ -46,25 +47,25 @@ impl VFileOps for Dipsw {
         &self,
         _: &VFile,
         com: IoctlCom,
-        mut data: &mut [u8],
+        data: &mut [u8],
         _: &Ucred,
         _: &VThread,
     ) -> Result<(), Box<dyn Errno>> {
         match com {
-            COM1 => todo!("dipsw ioctl 0x80028801"),
-            COM2 => todo!("dipsw ioctl 0x80028802"),
-            COM3 => todo!("dipsw ioctl 0xc0088803"),
-            COM4 => todo!("dipsw ioctl 0x80108804"),
-            COM5 => todo!("dipsw ioctl 0x80108805"),
-            COM6 => {
+            Self::COM1 => todo!("dipsw ioctl 0x80028801"),
+            Self::COM2 => todo!("dipsw ioctl 0x80028802"),
+            Self::COM3 => todo!("dipsw ioctl 0xc0088803"),
+            Self::COM4 => todo!("dipsw ioctl 0x80108804"),
+            Self::COM5 => todo!("dipsw ioctl 0x80108805"),
+            Self::COM6 => {
                 //todo write the correct value if unk_func1() = false and
                 // unk_func2() = true
-                data.write_i32::<LittleEndian>(false as i32).unwrap();
+                data.copy_from_slice(&(false as i32).to_le_bytes());
```

Comment:
Does the PS4 expect this value to be native endian? If yes better to write this as a native endian so in the future when we ported to big-endian it will be working correctly.

Reply:
So all the other times we call to_le_bytes, it should be to_ne bytes too?

Reply:
If the PS4 treat it as a native when reading or writing then yes.

Reply:
Ok